src/app/client/src/app/modules/shared/services/config/url.config.json

```diff
@@ -184,7 +184,8 @@
       "GET_DETAILS": "course/v1/batch/read",
       "GET_PARTICIPANT_LIST": "course/v1/batch/participants/list",
       "GET_BATCHS": "course/v1/batch/list",
-      "REMOVE_USERS": "course/v1/batch/user/remove"
+      "REMOVE_USERS": "course/v1/batch/user/remove",
+      "TERMS_AND_CONDITIONS": "https://docs.google.com/document/d/1QLtCVlHR_AELMmLDONxSekE__M-4sZaSe8pzpBj3g0U/edit"
```

Comment:
This is not the actual URL. This will vary for every environment. Do it similar to groups